### ISSUE TITLE
"Copy to clipboard" button missing: give it an accessible name

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#128]
From NYU A11y review: 
The "Copy to Clipboard" button on the individual result page is announced only as "button" - without an accessible name.

See: https://www.w3.org/TR/WCAG20/#ensure-compat-rsv

**Comment by lhenze** [comment]
PR is here https://github.com/inveniosoftware/invenio-app-rdm/pull/1123

**Comment by lhenze** [comment]
PR merged: https://github.com/inveniosoftware/invenio-app-rdm/commit/3d0523ed5ada73cf15c7dbe34633914e44ec9c92
